RELEASING.md

```diff
@@ -1,36 +1,237 @@
 
-# prepare the release
+# did you remember to setup your bintray credentials?
```

Comment:
I do find Brian's instructions on "Create Bintray.com Account and Get Access to nordstromoss" and "Setup Bintray Credentials" to be quite valuable. It was very helpful for me to follow. Can we please replace your "did you remember to setup your bintray credentials" with Brian's first two sections? https://confluence.nordstrom.net/display/ENGP/XIO%3A+How+to+Publish+New+Version

Reply:
Nvm, I'm deleting the above remark cuz it is too company-specific.